make/test.mk

```diff
@@ -294,7 +295,7 @@ ifeq ($(SET_IMAGE_NAME),)
             endif
         else
 			# if is running in CI than we expect that it's PR for toolchain-e2e repo (none of the images was provided), so use name that was used by openshift-ci
-			$(eval IMAGE_NAME := registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:${REPO_NAME})
+			$(eval IMAGE_NAME := default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/${OPENSHIFT_BUILD_NAMESPACE}/stable:${REPO_NAME})
         endif
     else
 		# an image name of the other operator was provided, then we don't have anything built for this one => use image built from master
```

Comment:
Shouldn't we update the line above too?

Reply:
yes, of course :-)

Reply:
wait, which line? I thought that you were referring to this change: https://github.com/codeready-toolchain/member-operator/pull/135/commits/42bace89adafd6c38feae0d44614fa59dc5d6fb3